ibm/resource_ibm_is_floating_ip.go

```diff
@@ -294,6 +294,8 @@ func fipGet(d *schema.ResourceData, meta interface{}, id string) error {
 	target, ok := floatingip.Target.(*vpcv1.FloatingIPTarget)
 	if ok {
 		d.Set(isFloatingIPTarget, target.ID)
+	} else {
+		d.Set(isFloatingIPTarget, "")
```

Comment:
why we need to set to ""

Reply:
when floating ip target is set initialli and then removed, we don't receive FLoatingIPTarget object, so if we don't set "", old target value will still be there. so setting "" if we don't receive target value